server/zanata-war/src/main/webapp/WEB-INF/layout/version/languages-tab.xhtml

```diff
@@ -201,7 +201,7 @@
           <ul class="dropdown__content js-dropdown__content" role="content"
             aria-labelledby="dropdownContent">
             <ui:fragment
-              rendered="#{versionHomeAction.zipFileDownloadAllowed}">
+              rendered="#{versionHomeAction.zipFileDownloadAllowed and versionHomeAction.getDocuments().size() > 0}">
```

Comment:
Side note: you can use ...` and !versionHomeAction.getDocuments().isEmpty()` too.